script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -549,7 +549,8 @@ The Dead that Arcueid took down earlier was nothing more than a 'servant',
 [sha:c06cc98ccf51f55304c457ea093c167e5e796138]{
 -- Page 552, Offset 17952.
 -- Ⅰ階梯なら自動人形にすぎないけど、Ⅲ階梯なら脳は生きているし。彼等は自分がどういう生き物になったのかを理解した上で吸血鬼に従っている。自分から進んで人間たちを殺して、親基に血を送っているのよ」
-A Rank I Dead is nothing more than an automaton, but a Rank III one still has a functioning brain.
+//typemoon.fandom.com/wiki/Familiar#Automata
+A Rank I Dead is nothing more than an %{i}automata%{/i}, but a Rank III one still has a functioning brain.
```

Comment:
revert. automaton _is_ the singular form of the plural automata.

Reply:
The reason as to why "automata" seems insisted upon here is because the ruby text given for 自動人形 in the Japanese text is オートマタ (automata). I think its reasonable to keep it grammatically correct for English though. So automaton it is.